rsciio/tests/data/topspin/create_topspin_data.py

```diff
@@ -0,0 +1,235 @@
+# -*- coding: utf-8 -*-
```

Comment:
@ericpre These test files are now downsized versions of the originals, not brand new files. I agree, much better and easier to follow.

Let me know if you are happy with these. I have not been able to collect un-downsized files still, our setup is currently not available to us unfortunately. It might be a few months before I can get intentionally tiny datasets that don't require downsizing.

Reply:
Okay, if this is not possible to generate these dataset, then it is fine to keep the existing files.